### PR TITLE
Fix static scan issues

### PR DESCRIPTION
**Pull request by xianju6x**

### CODE REVIEW

File: peeknpoke.c [hunk 1]
```diff
@@ -166,7 +166,9 @@ static int process_i2c_args(int argc, char **argv)
 		printf("\n***Invalid I2C operation\n");
 		status = -1;
 	}
-
+    if (values) {
```
Comment (jingdlu) [inline peeknpoke.c]: Please keep the same indent.

File: msr.c [hunk 2]
```diff
@@ -89,6 +89,6 @@ int msr_reg_write(int cpu, unsigned int reg_offset, uint64_t value)
 	}
 
 exit:
-	close(fd);
+    if (fd >= 0) close(fd);
```
Comment (jingdlu) [inline msr.c]: Please keep the same indent.

File: msr.c [hunk 3]
```diff
@@ -51,7 +51,7 @@ uint64_t msr_reg_read(int cpu, unsigned int reg_offset, int print_enabled)
...
```
Comment (jingdlu) [inline msr.c]: Please keep the same indent.

**Review by jingdlu — APPROVED** [review]
LGTM.

**Review by kwang13 — APPROVED** [review]

**Pull request merged**